### PR TITLE
Offer changes in README, and change the docker-compose file, to enabl…

### PR DESCRIPTION
**Pull request by eyalb-inabit**
…e quick setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
+
+2. In that account, there must be 2 existing users, with the following roles:
+  A. `apiAdmin` user.
+  B. `apiSigner` user.
+  
+This users should be  provided by our representatives.
+
+3. You have received a *login token* that is related to an "apiSigner" user. This token should have been provided to you by our representatives.
+ 
+4. Please verify that host can access `api.inabit.com`  server (whitelisted IPs and ports).
```
Comment (Ori-b) [inline README.md:45]: Check whitespaces:
" ... .com'     server"

File: README.md [hunk 2]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
+# clone this repository
+git clone https://github.com/In-a-bit/inabit-remote.git
+cd inabit-remote
+# create 2 needed directories
+mkdir refresh
+mkdir dat
+# save the login token in file.
+# in the next line, replace <login token> with the token you got.
+echo "<login token>" > refresh/r.dat
+touch dat/k.dat
```
Comment (Ori-b) [inline README.md:65]: k.dat is suppose to be created on the fly.

File: README.md [hunk 3]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
+# and copy the Pairing code 
+{
+  level: 'info',
+  message: 'Pairing code: <COPY THIS CODE>',
+  metadata: { timestamp: '2024-02-21T08:54:17.560Z' },
+  timestamp: '2024-02-21T08:54:17.560Z'
+}
+```
+
+**Send this code to you owner.**.
```
Comment (Ori-b) [inline README.md:109]: Pass this code

File: README.md [hunk 4]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
 ## Deployment
 
 From within the root directory `inabit/inabit-remote`:
 
  ### Set Login Token
  
- The login token is approver's token valid for 30 days,
- 
- allowing authentication with Inabit.
+ The login token is approver's token valid for 30 days, allowing authentication with Inabit.
```
Comment (Ori-b) [inline README.md:129]: is an approver's token that is valid for 30 days

**Review by Ori-b — APPROVED** [review]
Great Improvements !!!
quick setup is marvelous 
please refer to comments

File: apps/approver/.env.example [hunk 5]
```diff
@@ -12,4 +12,8 @@ VALIDATION_RETRY_INTERVAL_MINUTES=3
 VALIDATION_RETRY_MAX_COUNT=10
 LOGIN_TOKEN_FILE_PATH=refresh
 LOGIN_TOKEN_FILE_NAME=r.dat
-REFRESH_LOGIN_TOKEN_IN_MINUTES=21600 # 15 days in seconds
\ No newline at end of file
+REFRESH_LOGIN_TOKEN_IN_MINUTES=21600 # 15 days in seconds
+#Uncomment next line - only if you want to bypass the validation mechnisem.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply apps/approver/.env.example:16]: ok. removed. I just leave the allowed values.

File: README.md [hunk 6]
```diff
@@ -177,7 +267,7 @@ In order to handle transaction approvals:
             transactionId: string;
             transactionType: string;   // Withdrawal / Deposit / Swap
             initiatorId: string;
-            organizationId: string;
+            organizaationId: string;
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:270]: fixed.

File: README.md [hunk 7]
```diff
@@ -150,20 +238,22 @@ Within the flow interaction with Inabit,
...
-
+  
    1.  The external validation endpoint is expected to expose a 'post' http endpoint for validation purpose. 
    
       *connectivity to the endpoint must be enabled.
 
       ```env
                  
-            VALIDATION_CALLBACK_URL=http://koko.com/validation
+            VALIDATION_CALLBACK_URL=http://example.com./validation
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:255]: done.

File: README.md [hunk 8]
```diff
@@ -150,20 +238,22 @@ Within the flow interaction with Inabit,
...
+            # replace the domain with the URL of your validation login implemented endpoint.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:256]: done.

File: README.md [hunk 9]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
 
 ## Functionality
 
-Upon running, the application must successfully initialize and communicate with Inabit. This involves several steps depending on the initiation state. If the 'signer' is approved, the handling of pending transaction flow may proceed.
+Upon running, the application must successfully initialize and communicate with Inabit. This involves several steps depending on the initiation state.
+After this **Approver app** is approved by the owner and pairing process is completed, than, the handling of pending transactions flow may proceed naturally.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:8]: fixed.

File: README.md [hunk 10]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
 
 ### First Run
 
-During the first run, a pairing process is initiated. This includes:
+There is difference between the first run of the **Approver app** and versus recurring runs.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:12]: ok, I moved this sentence above. I still think it needed, but I moved it above the header, in a place, which is more fluent IMO.

File: README.md [hunk 11]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
-1. Logging into Inabit using the provided login token.
+Assuming you received all the details from our representatives, and You start the **Approver app** docker as described, so then
+During the first run, a pairing process is initiated. The **Approver app** will do these steps:
+
+1. Logging into Inabit using the provided login token, that you will receive from our team.
 2. Validating the user's current state.
 3. Requesting a pairing token.
 4. Producing an encrypted key used for signing, saving it to a mapped volume.
-5. Sending pairing data to Inabit and requesting the 'Owner' to authorize the approver.
+5. Sending pairing data to Inabit and requesting the 'Owner' to authorize the approver. **After this step, you will need to get the the _pairing token_ from the docker logs, and to send it to the owner of your Inabit account**.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:21]: done.

File: README.md [hunk 12]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
 
-1. An initial login token of a valid user within Inabit with the role: 'apiSigner'.
-2. Enabled communication with the Inabit server (valid URL and whitelisted IPs and ports).
+1. You must have an active account at `inabit.com`.
+
+2. In that account, there must be 2 existing users, with the following roles:
+  A. `apiAdmin` user.
+  B. `apiSigner` user.
+  
+This users should be  provided by our representatives.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:41]: done.

File: README.md [hunk 13]
```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
...
+
+3. You have received a *login token* that is related to an "apiSigner" user. This token should have been provided to you by our representatives.
```
[the remark this answers is not in the document]
Comment (eyalb-inabit) [reply README.md:43]: done.

File: README.md (hunk 1 above)
Comment (eyalb-inabit) [reply README.md:45]: done.

File: README.md (hunk 2 above)
Comment (eyalb-inabit) [reply README.md:65]: removed.

File: README.md (hunk 3 above)
Comment (eyalb-inabit) [reply README.md:109]: done.

File: README.md (hunk 4 above)
Comment (eyalb-inabit) [reply README.md:129]: done.

File: docker-compose.prod.yml [hunk 14]
```diff
@@ -5,12 +5,11 @@ services:
...
     volumes:
       - ./dat:/app/apps/approver/dat
       - ./refresh:/app/apps/approver/refresh
-    ports:
-      - "3020:3020"
-    env_file:
-      - ./.env
\ No newline at end of file
+      - ./wallet:/app/apps/approver/wallet
+ #   environment:
```
Comment (Ori-b) [inline docker-compose.prod.yml:14]: Remove the comments ?

File: apps/approver/.env.example [hunk 15]
```diff
@@ -8,12 +8,13 @@ APPROVER_CREATOR_EMAIL=issuer@example.com
...
 VALIDATION_MOCK_SET_RESULT=rejected
+# allowed: approved / rejected / exception
 VALIDATION_RETRY_INTERVAL_MINUTES=3
 VALIDATION_RETRY_MAX_COUNT=10
 LOGIN_TOKEN_FILE_PATH=refresh
 LOGIN_TOKEN_FILE_NAME=r.dat
-REFRESH_LOGIN_TOKEN_IN_MINUTES=21600 # 15 days in seconds
-#Uncomment next line - only if you want to bypass the validation mechnisem.
-#VALIDATION_MOCK_SET_RESULT=rejected  # allowed: approved / rejected / exception
+REFRESH_LOGIN_TOKEN_IN_MINUTES=21600 # 15 days in minutes 21600
```
Comment (Ori-b) [inline apps/approver/.env.example:16]: remove the 21600

**Review by Ori-b — APPROVED** [review]
Great Improvements!
see 2 minor comments

**Pull request merged**